hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/views/overview/overview.tsx

```diff
@@ -302,6 +319,16 @@ export class Overview extends React.Component<Record<string, object>, IOverviewS
           <Col xs={24} sm={18} md={12} lg={12} xl={6} className='summary-font'>
             <OverviewCard loading={loading} title='Pending Deleted Keys Summary' data={deletePendingSummaryData} icon='delete' linkToUrl='/Om'/>
           </Col>
+          {scmServiceId &&
+              <Col xs={24} sm={18} md={12} lg={12} xl={6} className='summary-font'>
+                <OverviewCard title="Storage Container Manager" loading={loading} data={scmServiceData} icon='file-text'/>
+              </Col>
```

Comment:
```suggestion
              <Col xs={24} sm={18} md={12} lg={12} xl={6}>
                <OverviewCard title="SCM Service" loading={loading} data={scmServiceId} icon='file-text'/>
              </Col>
```

I think we can simply do these to standardize with other overview card (e.g. Volumes / Bucket). Similarly to OM service as well.